### PR TITLE
initial setup and 1st test.

### PR DESCRIPTION
**Pull request by AndreWillomitzer**
@kibertoad 
Fixes: #62 
Hi my dude I fixed all the merge issues. It also works with export default now.

It still is skipping the test and says 0 of 0 and a nice big green success message, so we need to fix that but at least it runs, no errors, and uses JS files.

Have a look when you get the chance <3

### CODE REVIEW

**Comment by kibertoad** [comment]
just got home, will take a look now!

**Comment by kibertoad** [comment]
@AndreWillomitzer Thank you for reminding me that Jest is actually mostly compatible with Jasmine, thanks to that fact I actually managed to trick existing tests into working with Jasmine as well, so we don't even need to duplicate anything :D.
Can you update GitHub Actions to execute karma tests in addition to jest ones?

File: taskTester.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { ToadScheduler, SimpleIntervalJob, Task } = require('toad-scheduler')
```
Comment (kibertoad) [inline taskTester.ts:1]: I think this file is no longer needed

Comment (AndreWillomitzer) [reply taskTester.ts]: @kibertoad well I tried to add to ci.yml but that didn't work... it fails all the CI tests now.

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
@AndreWillomitzer Thanks a lot!
If you still have some time, can you investigate why three LongIntervalJob tests I had to skip for Jasmine are failing? I suspect it has to do with difference in how mock timers work in jest and jasmine, but I didn't look much into it.

**Comment by AndreWillomitzer** [comment]
> @AndreWillomitzer Thanks a lot! If you still have some time, can you investigate why three LongIntervalJob tests I had to skip for Jasmine are failing? I suspect it has to do with difference in how mock timers work in jest and jasmine, but I didn't look much into it.
@kibertoad No problem honestly! It's a good learning experience for myself because all of this stuff is used in a dev job.

I'll have to see honestly, might be tight because we have finals coming up the next couple weeks :O 

I super appreciate you letting me work on this again btw because now I have a PR for my assignment due Nov 19th 🚀 

Thanks brother!

**Pull request merged**